monkey/monkey_island/cc/ui/src/services/configuration/propagation/propagation.js

```diff
@@ -11,14 +11,15 @@ const PROPAGATION_CONFIGURATION_SCHEMA = {
     'maximum_depth': {
       'title': 'Maximum scan depth',
       'type': 'integer',
-      'minimum': 1,
+      'minimum': 0,
       'default': 2,
       'description': 'Amount of hops allowed for the monkey to spread from the ' +
       'Island server. \n' +
       ' \u26A0' +
       ' Note that setting this value too high may result in the ' +
       'Monkey propagating too far, '+
-      'if the "Local network scan" is enabled'
+      'if "Local network scan" is enabled.\n' +
+      'If the value is set to 0, everything except the propagation phase will run.'
```

Comment:
```suggestion
      'if "Local network scan" is enabled.\n'
```
I would drop this. "Propagation phase" isn't well defined in the UI. You could Say "Setting this to 0 will disable all scanning and exploitation.